### ISSUE TITLE
How to add my own images and get the output?

### ISSUE THREAD

**Issue by alekhra** [msieg/deep-visualizer#10]

**Comment by aneeshnema** [comment]
I don't think that would be possible, as this project uses BigGAN and the output is pictures are generated based on the classes selected from the available(ImageNet).
I think if you want to generate some other class/object, you will have to train a custom GAN yourself and then you can use a method similar to the one that is used in this project.
